java/src/apps/startup/configurexml/StartupActionsPreferencesPanelXml.java

```diff
@@ -2,9 +2,11 @@
 
 import apps.StartupActionsManager;
 import apps.startup.StartupModel;
+import jmri.ConfigureManager;
 import jmri.InstanceManager;
 import jmri.configurexml.AbstractXmlAdapter;
 import org.jdom2.Element;
+import org.python.jline.internal.Log;
```

Comment:
Let me track down that import. What I'll need to figure is which import it should be using instead. Any clues??